data/bunrodea/bunrodea outfits.txt

```diff
@@ -0,0 +1,333 @@
+
+outfit "Dark Reactor"
+	category "Power"
+	cost 15022000
+	thumbnail "outfit/dark reactor"
+	"mass" 142
+	"outfit space" -142
+	"energy generation" 24.7
+	"heat generation" 28.6
+	"reactor upgrades" 30
+#	ammo "Reactor Overclocker"
+#	ammo "Reactor Limiter"
+	description `The term "dark star" can have different meanings, but one definition of it is a star whose core is formed around a large amount of dark matter. Such exotic stars are theorized by humanity to have possibly existed among the first generations of stars in the universe, with concentrations of dark matter acting as nucleation sites.`
+	description "	Bunrodean reactors are able to be upgraded by installing Reactor Overclockers or Limiters, which alter the performance of the reactor. Each reactor only has a limited number of slots for these upgrades, though."
+
+
+
+outfit "Lasher Pistol"
+	category "Hand to Hand"
+	cost 230000
+	thumbnail "outfit/lasher pistol"
+	"capture attack" 2.6
+	"capture defense" 1.4
+	description "During their wars with the Korath, it was not uncommon for Bunrodean vessels to be boarded by Korath raiders looking for technology and supplies to steal. As such, effective small arms weapons became an essential part of every Bunrodean crew to fight off any potential boarding parties."
+
+
+
+outfit "Chiisana Rift Thruster"
+	category "Engines"
+	cost 130000
+	thumbnail "outfit/chiisana rift thruster"
+	"mass" 11
+	"outfit space" -11
+	"engine capacity" -11
+	"thrust" 6.25
+	"thrusting energy" 1.15
+	"thrusting heat" 0.65
+	"flare sprite" "effect/bunrodea flare/large"
+		"frame rate" 1
+		"rewind"
+		"scale" 0.3
+	"flare sound" "ion tiny"
+	description "While the Korath favor raw power in their engines, the Bunrodea favor efficiency, opting to make use of engines that produce less thrust for significantly less heat produced."
+
+outfit "Nami Rift Thruster"
+	category "Engines"
+	cost 262000
+	thumbnail "outfit/nami rift thruster"
+	"mass" 19
+	"outfit space" -19
+	"engine capacity" -19
+	"thrust" 12.1
+	"thrusting energy" 2.3
+	"thrusting heat" 1.3
+	"flare sprite" "effect/bunrodea flare/large"
+		"frame rate" 1
+		"rewind"
+		"scale" 0.6
+	"flare sound" "ion small"
+	description "Although Rift engines are extremely heat efficient, their energy requirement is rather high compared to the thrust they produce, even when compared to terribly inefficient Korath engines."
+
+outfit "Ookii Rift Thruster"
+	category "Engines"
+	cost 512000
+	thumbnail "outfit/ookii rift thruster"
+	"mass" 32
+	"outfit space" -32
+	"engine capacity" -32
+	"thrust" 22.9
+	"thrusting energy" 4.4
+	"thrusting heat" 2.5
+	"flare sprite" "effect/bunrodea flare/large"
+		"frame rate" 1
+		"rewind"
+		"scale" 0.8
+	"flare sound" "ion medium"
+	description "Even though the energy drawn per unit thrust of Rift engines is poorer than that of Korath engines, the overall energy requirement of Rift engines is still far lesser by comparison."
```

Comment:
```suggestion
	description "Even though the energy drawn per unit thrust of rift engines is poorer than that of Korath engines, the overall energy requirement of rift engines is still far lesser by comparison."
```

Reply:
Okay, this is an interesting one. Is `Rift` a description of the technology (ion engine), or some marketing / company name?